### PR TITLE
Ticket8update votes

### PR DESCRIPTION
**Pull request by FimbulWinters**

### CODE REVIEW

**Comment by pkaodev** [comment]
Looks good!  We can add another test for the sad path:
-when the vote object has an incorrect body (inc_votes is missing, or is not a number).

Add another test for that and then submit another nchelp pr please.

File: __tests__/patch.test.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    });
+    test("Should return error 400 when given an invalid inc_votes object", () => {
+      return request(app)
+        .patch("/api/reviews/2")
+        .send({ inc_votes: "squirrel" })
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.message).toBe("Bad request!");
+        });
+    });
```
Comment (sirKevlar) [inline __tests__/patch.test.js:106]: One more test needed here. It's possible for the key 'inc_votes' to be misspelled

File: Controllers/patchController.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const { updateVotes } = require("../Models/patchModel");
+
+exports.patchReviewVotes = (req, res, next) => {
+  const { review_id } = req.params;
+  const body = req.body;
+
+  updateVotes(review_id, body)
+    .then((review) => {
+      res.status(202).send({ review });
```
Comment (callumu21) [inline Controllers/patchController.js:9]: Wrong status code here for a successful PATCH request.

**Review by callumu21 — APPROVED** [review]
This looks good with a small change to the status code for a PATCH request. You do have some merge issues to deal with before you can merge this into main though, so be careful that you don't lose anything handling the conflict.

**Pull request merged**